web/app/components/project/jira-widget.ts

```diff
@@ -75,7 +75,72 @@ export default class ProjectJiraWidgetComponent extends Component<ProjectJiraWid
    */
   protected get issueStatus() {
     if (this.issue && "status" in this.issue) {
-      return this.issue.status;
+      return this.issue.status as string;
+    }
+  }
+
+  /**
+   * The type of the issue, if it exists.
+   * Will exist if the issue is a full Jira issue.
+   * Determines if the type is shown in the widget.
+   */
+  protected get issueType() {
```

Comment:
Curious why we need these getters and can't just do conditionals in the template based on if the issue's properties are undefined or not?

Reply:
For some reason Glint has trouble when using union types like this

```
{{! this.issue: PickedJiraIssue | JiraIssue }}

{{#if this.issue.issueType}}
   {{!-- if this passes, it means this.issue is type: JiraIssue --}}
   {{!-- however Glint thinks it still might be a PickedJiraIssue --}}
   
   {{this.issue.issueType}}
   {{!-- this gets underlined by Glint --}}
{{/if}}
```